### DISCUSSION TITLE
Can not add argument "profile-directory" to open an existing profile

### DISCUSSION THREAD

**Discussion by tranngocminhhieu** [seleniumbase/SeleniumBase#2331] [Q&A] [answered]
Hello everyone,

I try to log in to Google with a specific profile. But SB always opens profile "Default" in user-data-dir, how can I add argument "profile-directory" like selenium options?

Thank you!

**Comment by anthonyg45157** [comment]
Edit: Read the official answer... mine isn't the greatest it seems, still learning 

Hello there. Just a newb that is willing to help here..

Kind of a round about method but  I used a new install of chrome with only 1 newly created profile and copied only that profile over to my user data dir, SB used this one by default and it worked for me.

if you have multiple you might need a little more assistance here

**Comment by mdmintz** [comment, marked as answer]
If you need multiple profiles, set a different `user_data_dir` for each one that you want. Only the `Default` profile is supported.

If you're setting the `user_data_dir` while using UC Mode, then make sure you read https://github.com/seleniumbase/SeleniumBase/discussions/2118#discussioncomment-7029201 first. (Don't mix a regular `user_data_dir` with one that is used by UC Mode Chrome, as the two formats are not compatible.)
